src/cmd/hotspots/add.rs

```diff
@@ -1,106 +1,480 @@
+
+            let ix = program
+                .request()
+                .args(helium_entity_manager::instruction::IssueDataOnlyEntityV0 {
+                    args: IssueDataOnlyEntityArgsV0 {
+                        entity_key: entity_key.clone(),
+                    },
+                })
+                .accounts(issue_entity_accounts)
+                .instruction(compute_ix)
+                .instructions()
+                .unwrap();
+
+            let mut tx = SolanaTransaction::new_with_payer(&ix, Some(&keypair.pubkey()));
+            let blockhash = program.rpc().get_latest_blockhash()?;
+
+            tx.try_partial_sign(&[Rc::as_ref(&keypair)], blockhash)
+                .map_err(|e| anyhow!("Error while signing tx: {e}"))?;
+
+            let serialized_tx = hex::encode(&bincode::serialize(&tx).unwrap());
+            let sig = add_gateway_txn.gateway_signature.clone();
+            add_gateway_txn.gateway_signature = vec![];
+
+            let mut buf = vec![];
+            add_gateway_txn.encode(&mut buf).unwrap();
+
+            // verify the base64 transaction with the ecc-sig-verifier
+            let req_client = reqwest::blocking::Client::new();
+            let response = req_client
+                .post(format!("{}/verify", ecc_verifier_url))
+                .json(&VerifyRequest {
+                    transaction: &serialized_tx,
+                    msg: &hex::encode(&buf),
+                    signature: &hex::encode(sig),
+                })
+                .send()
+                .map_err(|e| anyhow!("Error while sending request: {e}"))?
+                .json::<VerifyResponse>()
+                .map_err(|e| anyhow!("Error while parsing response: {e}"))?;
+            let raw_signed_tx = hex::decode(response.transaction).unwrap();
+            let signed_tx: SolanaTransaction = bincode::deserialize(&raw_signed_tx).unwrap();
+            println!("Transaction signed: {:?}", signed_tx.is_signed());
+
+            program
+                .rpc()
+                .send_and_confirm_transaction_with_spinner(&signed_tx)
+                .unwrap();
+        } else {
+            println!("Entity already issued");
         }
+
+        println!("Onboarding hotspot");
+
+        let kta_acc = program
+            .account::<KeyToAssetV0>(key_to_asset)
+            .map_err(|e| anyhow!("Failed to get key_to_asset account: {e}"))?;
+        let onboard_accounts = construct_onboard_iot_accounts(
+            &program,
+            Pubkey::from_str(HNT_MINT).unwrap(),
+            &entity_key,
+        )
+        .map_err(|e| anyhow!("Failed to create onboard iot accounts: {e}"))?;
+
+        let req_client = reqwest::blocking::Client::new();
+
+        let get_asset_response = req_client
+            .post(program.rpc().url())
+            .header("Cache-Control", "no-cache")
+            .header("Pragma", "no-cache")
+            .header("Expires", "0")
+            .json(&JsonRpcRequest {
```

Comment:
There's already the jsonrpc crate that helps with all this... no need to hardcode all this in